### PR TITLE
Rename the requirements

### PR DESCRIPTION
**Pull request by queenypingcap**

### CODE REVIEW

**Comment by queenypingcap** [comment]
@sunhao2017 PTAL.

**Comment by andelf** [comment]
oops, you can use git rm and git mv to do this.
instead of copy & paste content.

or commit diff log will be messed up.

**Comment by queenypingcap** [comment]
Got it. Thanks for the reminder!

File: op-guide/recommendation.md [hunk 1]
```diff
@@ -1,51 +1,74 @@
...
+| CentOS                   | 7.3 及以上   |
+| Oracle Enterprise Linux  | 7.3 及以上   |
+| Ubuntu LTS               | 14.04 及以上 |
 
-* OLTP 线上业务强烈建议使用 SSD 硬盘，否则在一些场景下会有性能问题。
-* TiKV 硬盘大小建议不要超过500G（防止硬盘损坏时，数据恢复耗时过长）
+> 注：Oracle Enterprise Linux 只支持 Red Hat 兼容内核 (RHCK) , 不支持 Oracle Enterprise Linux 提供的 Unbreakable Enterprise Kernel。
 
-## 整体部署方案
+> 注：以上所支持的 Linux 操作系统包括部署和运行在物理服务器以及 VMWare, KVM, XEN 主流虚拟化环境。
```
Comment (andelf) [inline op-guide/recommendation.md]: ref: https://www.vmware.com/help/trademarks.html

Its trademark is `VMware`, not `VMWare`

**Pull request merged**